rice-middleware/impl/src/main/java/edu/arizona/rice/kim/service/UaUiDocumentService.java

```diff
@@ -3,7 +3,7 @@
  * Copyright 2009 State of Arizona Board of Regents
  *
  */
-package edu.arizona.kim.service.impl;
+package edu.arizona.rice.kim.service;
```

Comment:
Thank you for catching this, Natalia. This code was migrated from rice-2.1-ua-development originally that I didn't look into its package and name. I made recommended code change and will retest, commit and squash.